src/transport/FabricTable.cpp

```diff
@@ -134,9 +134,9 @@ CHIP_ERROR FabricInfo::FetchFromKVS(PersistentStorageDelegate * kvs)
 
     SuccessOrExit(err = kvs->SyncGetKeyValue(key, info, infoSize));
 
-    mNodeId     = Encoding::LittleEndian::HostSwap64(info->mNodeId);
+    mOperationalId.SetNodeId(Encoding::LittleEndian::HostSwap64(info->mNodeId));
+    mOperationalId.SetFabricId(Encoding::LittleEndian::HostSwap64(info->mFabricId));
```

Comment:
The operational ID must also include the root public key, since fabric ID is not unique

Reply:
I agree. I'd prefer to do this as part of a separate PR, where we bring in support for multiple fabrics.
This PR is mostly focused on removing incorrect process of assignment of node ID.